packages/wasm-dpp/test/unit/identity/stateTransition/IdentityTopUpTransition/applyIdentityTopUpTransitionFactory.spec.js

```diff
@@ -27,11 +27,11 @@ describe('applyIdentityTopUpTransitionFactory', () => {
 
   beforeEach(async function beforeEach() {
     stateRepositoryMock = createStateRepositoryMock(this.sinonSandbox);
-    stateRepositoryMock.fetchIdentityBalanceWithDebt.returns(0);
+    stateRepositoryMock.fetchIdentityBalanceWithDebt.resolves(1);
```

Comment:
It's probably failing here due to `is_falsy` check. We can bring it back.

Reply:
Fixed